src/extensions/amazon-sales/release-notes/index.md

```diff
@@ -21,6 +21,32 @@ The release notes include:
 
 See [Upcoming Releases](https://devdocs.magento.com/release/) for versioning, support, and compatibility.
 
+### v4.2.0
+
+Amazon Sales Channel 4.2.0 is compatible with versions 2.3.x but only supported in versions 2.4.x of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}. If you have a previous Amazon Sales Channel version installed and attempt to update your Magento to version 2.4.0, you will be prompted to update the extension before you can complete the Magento update.
```

Comment:
```suggestion
Amazon Sales Channel 4.2.0 is compatible with versions 2.3.x but only supported 2.4.x versions of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}. If you have a previous Amazon Sales Channel version installed and attempt to update your Magento to version 2.4.0, you will be prompted to update the extension before you can complete the Magento update.
```

Reply:
changed "in" to "for", removal makes it awkward